security/fma-operator-fee.md

```diff
@@ -127,4 +127,10 @@ Below are references for this project:
 Below is what needs to be done before launch to reduce the chances of the above failure modes occurring, and to ensure they can be detected and recovered from:
 
 - [ ] Coordinate with wallet providers to update their fee estimation logic
-- [ ] Implement automated monitoring on dabase growth rate
\ No newline at end of file
+- [ ] Implement automated monitoring on dabase growth rate
+
+## Audit Requirements
+
+An audit has not been deemed necessary for the relatively simple changes to the SystemConfig contract, which has been reviewed by Base internal security team.
+
+Indeed, the only addition is the `setOperatorFeeScalars` function, which is a simple setter function that updates the operator fee parameters and trigger an event. This function is callable by the SystemConfig owner only.
```

Comment:
Can we be a little more rigid here? "Simple" is subjective, which quadrant does this feature fall into in the audit framework? https://gov.optimism.io/t/op-labs-audit-framework-when-to-get-external-security-review-and-how-to-prepare-for-it/6864
![image](https://github.com/user-attachments/assets/15e0e26d-b152-4078-918a-895c13e95ddc)

Reply:
@mds1 I added a commit, is it ok for you?

Reply:
Thanks!